spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -56,6 +56,7 @@
       let(:variant) { product_tax.variants.first }
 
       before do
+        Spree::Config.set allow_backorders: false
```

Comment:
@enricostano I tried yesterday and didn't work. You don't see the code because I am rebasing to make commits tidy